habitat_baselines/il/models/models.py

```diff
@@ -348,3 +348,357 @@ def forward(
         scores = self.classifier(mul_feats)
 
         return scores, att_probs
+
+
+class MaskedNLLCriterion(nn.Module):
```

Comment:
Return types missing for these functions as well as function args for a lot of things